win32ss/user/ntuser/ghost.c

```diff
@@ -264,12 +270,10 @@ GhostThreadProc(PVOID Param)
 Quit:
     RtlFreeHeap(GlobalUserHeap, 0, gGhostInfo);
 
-    NtSetEvent(gGhostInfo->GhostQuitEvent, NULL);
+    KeSetEvent(gGhostInfo->GhostQuitEvent, 1, TRUE);
```

Comment:
Why "1"? Shouldn't it be IO_NO_INCREMENT instead?

Reply:
OK.